services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/dao/MacRangeRepository.java

```diff
@@ -75,22 +76,38 @@ public MacRange findItem(String rangeId) throws CacheException {
         return hashMap;
     }
 
+    /**
+     * add a MAC range to MAC range repository
+     *
+     * @param macRange MAC range
+     * @return void
+     * @throws Exception Db or cache operation exception
+     */
     @Override
-    public void addItem(MacRange macRange) throws CacheException {
-        try {
+    public void addItem(MacRange macRange) throws Exception {
```

Comment:
Revert it back to CacheException as the following catch only throws CacheException.

Reply:
Revert is done.